gradle/libs.versions.toml

```
@@ -4,9 +4,9 @@ kotlin = "2.1.21"
 kotlinx-html = "0.12.0"
 kotlinx-io = "0.7.0"
 coroutines = "1.10.2"
-atomicfu = "0.27.0"
+atomicfu = "0.28.0"
 serialization = "1.8.1"
```

```suggestion
serialization = "1.9.0"
```

`kotlin.time` is only support from 1.9.0, so I think should be bumped.

---

Sorry was too late to remove this comment after our conversation on Slack 👍